whylog_vim/utils.py

```diff
@@ -0,0 +1,14 @@
+from whylog_vim.consts import LineNames
+
+
+def get_parser_name(id_):
```

Comment:
Please rename `_id` -> `something_id` where something could be parser or line...